dj_swud/app/controllers/dj_swud_controller.rb

```diff
@@ -0,0 +1,67 @@
+
+class DjSwudController < ApplicationController
+  def index
+  end
+
+  def random_word
+  	response = 'http://randomword.setgetgo.com/get.php'
+  end
+
+  def show
+  end
+  
+  # i need to send a get request to this url: http://swapi.co/api/people/1/ in order to access the 
+  # person "luke skywalker". each person has a specified number, ranging from 1 to 86. there are 86 people
+  # in this api.
+
+  # i then need to pull out the value(luke skywalker) for the "name" key, which is in position[0] of the "results" array in the 
+  # parsed json. the json is a hash with 4 keys. the 'results' key contains arrays for each person. each
+  # array contains hashes, but im interested in only the "name"key and "lukeskywalker" value, for the example 
+  # of trying to pullout the name "luke skywalker" for the next step. 
+  # I then need to check the "luke skywalker" string for any spaces, indicating a last name, and then
+  # get rid of the last name so that only "Luke" is available to append the random word generated from my
+  # other api. 
+  #my biggest problem is trying to randomly generate a number in the url in order to get this thing 
+  #to be anywhere near what i planned as a mashup project. i had an idea that was way beyond 
+  #skill level in regards to bringing it to any possible functionality. I AM SCREWED!!!! 
+
+  def person
+
+  	person_one = Swapi.get_person 1      #returns the following STRING:
```

Comment:
random_number = rand(86) + 1 #this makes a random number between 1 and 86
person = Swapi.get_person random_number